### DISCUSSION TITLE
Cashier Paddle

### DISCUSSION THREAD

**Discussion by driesvints** [HassanZahirnia/laravel-package-ocean#191] [Package Suggestions]
Saw https://github.com/laravel/cashier-paddle wasn't in here yet. Also current "Cashier" can probably be renamed to "Cashier Stripe" to prevent confusion.

Just while I'm in here, I noticed Pail being the only official package with "Laravel" in front of it. Can probably just be named "Pail" like the other first party packages.

**Comment by HassanZahirnia** [comment]
Done! Thanks a lot again for the edit suggestions! 😇
